lib/inspec/resources/virtualization.rb

```diff
@@ -253,7 +304,40 @@ def collect_data_linux
       return if detect_openvz
       return if detect_openstack
       return if detect_parallels
-      # TODO: where is vmware?
+      return if detect_vmware
+      return if detect_hyperv
+    end
+
+    def windows_computer_system
+      script = "Get-WmiObject -Class Win32_ComputerSystem -ErrorAction Stop | ConvertTo-Json"
+      cmd = inspec.powershell(script)
+
+      JSON.parse(cmd.stdout)
+    rescue JSON::ParserError => _e
+      nil
+    end
+
+    def collect_data_windows
+      # This avoids doing multiple detections in a single test
+      return unless @virtualization_data.empty?
+
+      case windows_computer_system["Model"]
+      when "Virtual Machine"
+        @virtualization_data[:system] = "hyper-v"
```

Comment:
You may want to look at pulling the full DMI decode logic out of Ohai for use here. It would give you pretty robust cross platform virtualization detection.